fs/fs.go

```diff
@@ -165,11 +171,19 @@ func (self *RealFsInfo) addDockerImagesLabel(context Context) {
 		dockerPaths := getDockerImagePaths(context)
 
 		for src, p := range self.partitions {
-			self.updateDockerImagesPath(src, p.mountpoint, dockerPaths)
+			self.updateContainerLabelImagesPath(LabelDockerImages, src, p.mountpoint, dockerPaths)
```

Comment:
See my other pr i do that there.
On Feb 10, 2016 3:49 PM, "Vish Kannan" <notifications@github.com> wrote:

> In fs/fs.go
> <https://github.com/google/cadvisor/pull/1104#discussion_r52546372>:
>
> > @@ -165,11 +171,19 @@ func (self *RealFsInfo) addDockerImagesLabel(context Context) {
> >  		dockerPaths := getDockerImagePaths(context)
> >
> >  		for src, p := range self.partitions {
> > -			self.updateDockerImagesPath(src, p.mountpoint, dockerPaths)
> > +			self.updateContainerLabelImagesPath(LabelDockerImages, src, p.mountpoint, dockerPaths)
>
> It is kind of weird to make addDockerImagesLabel update labels for rkt as
> well. This logic might require a more thorough refactoring.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1104/files#r52546372>.
>